airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/utils.py

```diff
@@ -321,4 +327,9 @@ def transform_data(records: List) -> Iterable[Mapping]:
         if "variables" in record:
             record = transform_variables(record)
 
+        # replace `pivot` with `_pivot`, to allow redshift normalization,
+        # since `pivot` is a reserved keyword for Redshift.
+        # on behalf of https://github.com/airbytehq/airbyte/issues/13018
+        record = transform_col_names(record, ["pivot"])
```

Comment:
It's look like `["pivot"]` can be as variable (e.g. `destination reserved names`)

Reply:
Done, good catch.